artifacts/argocd-apps/sample-apps/envs/prod/templates/team-carmen.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
```

Comment:
Why is Team Carmen needed? he is my skip level btw.

Reply:
team-carmen removed